package.json

```diff
@@ -43,13 +43,13 @@
     "typescript": "4.1.3"
   },
   "peerDependencies": {
+    "@angular-eslint/template-parser": ">=2.0.2 <13.0.0",
```

Comment:
Could you move these versions as they were prevoiusly to keep the git history per line? 🤔

Reply:
Running yarn commands automatically sorts the imports. 

I would leave it like this as otherwise we will need to do this on each MR. If needed i can split this into multiple commits if you are worried about git history.

Reply:
Ok, so we didn't have them alphabetically ordered before? I hope this won't change randomly on each change...